tests/fv3_conf/cpld_control_run.IN

```diff
@@ -42,6 +42,7 @@ cp    @[INPUTDATA_ROOT]/${FV3_DIR}/INPUT/grid_spec.nc ./INPUT/@[ATMRES]_mosaic.n
 
 # MOM6 fixed input
 cp    @[INPUTDATA_ROOT]/MOM6_FIX/@[OCNRES]/* ./INPUT
+cp /scratch1/NCEPDEV/climate/Jessica.Meixner/MOM6settings/global-workflow/fix/mom6/500/oceanda_zgrid_25L.nc ./INPUT
```

Comment:
I assume this line is not needed once the oceanda_zgrid_25L.nc is copied to input directory.

Reply:
Yes, that is correct.

Reply:
I'm not seeing the file in this location: ``/scratch1/NCEPDEV/climate/Jessica.Meixner/MOM6settings/``

Reply:
@DeniseWorthen I thought I put in the actual fix file location, not my personal link to the global-workflow fix.  I'll update this now here and on the PR template.

Reply:
This is the correct location: /scratch1/NCEPDEV/global/glopara/fix/mom6/20220805/500/oceanda_zgrid_25L.nc

Reply:
Ok, thanks.